workflows/qc/quality-check-standard.wdl

```diff
@@ -208,35 +204,80 @@ workflow quality_check {
             max_retries=max_retries
         }
     }
+
+    call picard.mark_duplicates as markdups { input:
+        bam=quickcheck.checked_bam,
+        create_bam=mark_duplicates,
+        max_retries=max_retries
+    }
+    if (mark_duplicates) {
+        call mark_duplicates_post.markdups_post { input:
+            markdups_bam=select_first([markdups.duplicate_marked_bam, "undefined"]),
```

Comment:
Should this be wrapped for consistency?

Reply:
Good question and IDK the answer. I think "consistency" is a valid reason to wrap this. However it's only 85 characters wide (below the 90 character limit). Do we encourage wrapping for consistency/style or encourage using the character limit as a guide?

I could go either way.

Reply:
I'd agree with that. Will make the change.